streams/src/main/java/org/apache/kafka/streams/processor/TimestampExtractor.java

```diff
@@ -26,10 +26,10 @@
 public interface TimestampExtractor {
 
     /**
-     * Extracts a timestamp from a message
+     * Extracts a timestamp from a message.
```

Comment:
Discussed offline, for now we do not enforce any semantics in the `TimestampExtractor`. We may want to expand this in the future though, so that this function returns a `Timestamp` object which contains the `long` value as well as the `TimeUnit`, and then in the DSL users will not need to do the manual conversion but just do `(5, TimeUnit.MINUTE)`.